web/themes/custom/pineapple/components/_comments.twig

```diff
@@ -89,23 +89,23 @@
     author_id == 1 ? 'admin-comment',
   ]) %}
 
-  <footer class="comment-meta">
-    {# User picture will be added in a later task.  #}
-{#    {{ user_picture }}#}
+  <div class="comment-info">
+    {{ user_picture }}
+    <footer class="comment-meta">
+      {% set author_name = comment.getAuthorName() %}
+      <p{{ submitted_attributes }}>{{ author_name }} says</p>
+      <p class="created">{{ created }}</p>
```

Comment:
Looks like the date format being used is the american standard please can you update the format here: admin/config/regional/date-time/formats/manage/short to use the british format.

Reply:
Done